plugin.js

```diff
@@ -38,6 +38,7 @@ function fastifyCookieClearCookie (reply, name, options) {
 
 function onReqHandlerWrapper (options) {
   return function fastifyCookieOnReqHandler (fastifyReq, fastifyRes, done) {
+    fastifyReq.cookies = {} // New container per request. Issue #53
```

Comment:
If we put this line after `if (cookieHeader) {` are not avoiding to allocate an empty object each request without cookies?

Reply:
That would still leave the old container around between requests.